specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/examples/inference/createOrUpdateACIOnlineEndpoint.json

```diff
@@ -0,0 +1,82 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "onlineEndpointName": "testendpoint",
+    "api-version": "2020-09-01-preview",
+    "properties": {
+      "location": "westus2euap",
+      "tags": {},
+      "kind": "ACI",
+      "properties": {
+        "description": "This is a test endpoint",
+        "properties": {},
+        "trafficRules": {},
+        "computeConfiguration": {
```

Comment:
fix spec to be base class

Reply:
making ComputeConfiguration as base class does not work for lintdiff. [Build with errors](https://dev.azure.com/azure-sdk/public/_build/results?buildId=551837&view=logs&j=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&t=43324f1f-6724-5d61-785f-f0ac9fd256a1&l=51)

Reply:
figured out the issue with previous setup. Updated swagger.